nf_core/modules/_completion.py

```diff
@@ -0,0 +1,35 @@
+import sys
```

Comment:
as ususal the modules completion and the subworkflow completion are quite similar. can you combine them and maybe add to component_utils.py?

Reply:
Done